docker-build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+cd $(dirname $0)
+
+# create the container that will build the static 
+# assets
+
+# clean out the static folder
+rm -r static/*
+
+# make sure the build container is updated
+# the build container only includes the necessary 
+# system dependencies to build out the static assets
+sudo docker build -t kuma:builder -f Dockerfile-builder .
+
+# Build the static assets by volume mounting the 
+# current source code
+docker run -it --rm=true -v "$PWD:/app" kuma:builder \
+    sh -c './manage.py collectstatic --noinput && ./manage.py compilejsi18n'
```

Comment:
These need to be reversed now since `compilejsi18n` now creates files that `collectstatic` processes.